src/app/creator-profile-page/creator-profile-row/creator-profile-simple-card.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="d-flex js-creator-profile-top-card-container creator-profile__top-card-profile-row">
```

Comment:
The file is a copy of the part from the `creator-profile-top-card`. 

It can be reused there for optimization, but I didn't do that yet not to break existing logic